application-workloads/azure-gamedev/gamedev-vm/nestedtemplates/gamedev-vm.bicep

```diff
@@ -586,33 +643,39 @@ resource virtualMachine 'Microsoft.Compute/virtualMachines@2021-11-01' = {
   }
 }
 
-module remoteAccess 'remoteAccessExtension.bicep' = {
-  name: 'runRemoteAccess'
-  params: {
-    cmdGDKInstall: cmdGDKInstall
-    cmdTeradiciRegistration: cmdTeradiciRegistration
-    _artifactsLocation: _artifactsLocation
-    _artifactsLocationSasToken: _artifactsLocationSasToken
-    virtualMachineName: virtualMachine.name
-    remoteAccessTechnology: remoteAccessTechnology
-    location: location
-    fileShareStorageAccount: fileShareStorageAccount
-    fileShareStorageAccountKey: fileShareStorageAccountKey
-    fileShareName: fileShareName
-    p4Port: p4Port
-    p4Username: p4Username
-    p4Password: p4Password
-    p4Workspace: p4Workspace
-    p4Stream: p4Stream
-    p4ClientViews: p4ClientViews
+resource virtualMachine_GDVMCustomization 'Microsoft.Compute/virtualMachines/extensions@2019-12-01' = if (deployedFromSolutionTemplate) {
```

Comment:
why is this only done when deployed from marketplace?

Reply:
we have not yet moved these scripts to github, and instead they are now included directly within the VM image. Let me take this back under consideration, and we may decide to also include the files here.

Reply:
would like to take on adding those files in a separate PR and lock in these updates.